src/components/EditFreakForm/__test__/EditFreakForm.test.jsx

```diff
@@ -0,0 +1,204 @@
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.lastName]: 'Badea',
+    });
+  });
+
+  it('should trigger onChange when the email is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const emailInput = screen.getByTestId('email-input');
+
+    fireEvent.change(emailInput, { target: { value: 'badea@email.com' } });
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.email]: 'badea@email.com',
+    });
+  });
+
+  it('should trigger onChange when the description is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const descriptionInput = screen.getByTestId('description-input');
+
+    fireEvent.change(descriptionInput, { target: { value: 'I am Marian' } });
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.description]: 'I am Marian',
+    });
+  });
+  it('should trigger onChange when the description is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const roleInput = screen.getByTestId('role-input');
+
+    fireEvent.change(roleInput, { target: { value: 'Founder' } });
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.role]: 'Founder',
+    });
+  });
+
+  it('should trigger onChange when the level is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const levelInput = screen.getByTestId('level-input');
+
+    fireEvent.change(levelInput, { target: { value: 'Novice' } });
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.level]: 'Novice',
+    });
+  });
+
+  it('should trigger onChange when the norm is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const normInput = screen.getByTestId('norm-input');
+
+    fireEvent.change(normInput, { target: { value: 'Full time' } });
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.norm]: 'Full time',
+    });
+  });
+
+  it('should trigger onChange when the description is changed', () => {
+    render(
+      <EditFreakForm
+        onChange={ onChange }
+      />,
+    );
+
+    const descriptionInput = screen.getByTestId('description-input');
+
+    fireEvent.change(descriptionInput, { target: { value: 'Elm' } });
+
+    expect(onChange).toBeCalledWith({
+      ...FreakModelDefault,
+      [FreakModelKeys.description]: 'Elm',
+    });
+  });
+
+  it('should receive a freak and display its data', () => {
+    const freak = {
+      description: '',
+      email: '',
+      firstName: '',
+      lastname: '',
+      level: '',
+      norm: '',
+      role: '',
+      skills: '',
+    };
```

Comment:
You could have used the `FreakModelDefault` for this. Even better, use some data that reflects real life data, you wouldn't pass a freak that has all fields empty. Give him some actual data. Also, you can extract this somewhere outside and use it in the next test as well to avoid duplication.